contracts/PvpArena.sol

```diff
@@ -1092,4 +1092,9 @@ contract PvpArena is Initializable, AccessControlUpgradeable {
     {
         rankingPointsByCharacter[characterID] = newRankingPoints;
     }
+    /// @dev returns the amount of matcheable characters
+    function getMatchablePlayers(uint256 characterID) public view returns(uint){
```

Comment:
```suggestion
    function getMatchablePlayersCount(uint256 characterID) public view returns(uint){
```

This feels as if it would return a list of players :p